Composer/packages/lib/indexers/src/utils/dialogCheckUtil.ts

```diff
@@ -21,3 +22,15 @@ export function autofixReferInDialog(dialogId: string, content: string): string
     return content;
   }
 }
+
+export function getRecognizerType(recognizer): RecognizerType {
+  if (typeof recognizer === 'string') {
+    return 'DefaultRecognizer';
+  }
```

Comment:
We need to consider the custom recognizer case here

Reply:
FYI - useRecognizerConfig, ui-plugins/luis index.ts, ui-plugins/cross-train index.ts